pex/common.py

```diff
@@ -348,7 +348,7 @@ def __str__(self):
   def delete(self):
     shutil.rmtree(self.chroot)
 
-  def zip(self, filename, mode='wb'):
+  def zip(self, filename, mode='w'):
```

Comment:
opening a zip in non-binary mode strikes me as a bit odd.. what's the reasoning behind this?

Reply:
It's not even a thing! That was a bug. Zipfile only supports a subset of modes - this is not an open pass-thru.

Reply:
To be more clear, the one user passed `mode='a'`, avoiding the bug.